source/docs/contributing/wpilib/other-wpilib-versions.rst

```diff
@@ -0,0 +1,84 @@
+Installing Development Builds
+=============================
+
+This article contains instructions on building projects using a development build and a local WPILib build.
+
+.. note:: This only applies to Java/C++ teams.
+
+Development Build
+-----------------
+
+In order to build a project using a development build, find the ``build.gradle`` file and open it. Then, add the following code below the plugin section and replace ``YEAR`` with the year of the development version.
```

Comment:
I think this needs some introduction of what a development build is and what the caveats are, ie that every commit to master is built, and that sometimes (ie around beta) development builds won't work without the beta image.